readme_renderer/rst.py

```diff
@@ -14,20 +14,28 @@
 from __future__ import absolute_import, division, print_function
 
 import io
+import typing
 
 from docutils.core import publish_parts
+from docutils.nodes import colspec, image
 from docutils.writers.html4css1 import HTMLTranslator, Writer
 from docutils.utils import SystemMessage
 
 from .clean import clean
 
 
-class ReadMeHTMLTranslator(HTMLTranslator):
+class ReadMeHTMLTranslator(HTMLTranslator):  # type: ignore[misc] # docutils is incomplete, returns `Any` python/typeshed#7256 # noqa E501
 
     # Overrides base class not to output `<object>` tag for SVG images.
-    object_image_types = {}
-
-    def emptytag(self, node, tagname, suffix="\n", **attributes):
+    object_image_types = {}  # type: ignore[var-annotated] # intentnionally empty
```

Comment:
Out of curiosity, I looked at the [docutils source code](https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/docutils/writers/html4css1/__init__.py#l543). Looks like this could be:

```suggestion
    object_image_types: typing.Dict[str, str] = {}
```